metadata-service/factories/src/main/resources/application.yml

```diff
@@ -86,6 +86,9 @@ visualConfig:
   assets:
     logoUrl: ${REACT_APP_LOGO_URL:#{null}}
 
+trackingConfig:
```

Comment:
There is a tracking block already https://github.com/datahub-project/datahub/blob/master/metadata-service/factories/src/main/resources/application.yml#L45 which is being sent via gms config